app/assets/stylesheets/replies.scss

```diff
@@ -436,16 +488,20 @@ div.post-subheader { width: 100%; }
   font-size: 70%;
 }
 
-.reply-highlighted { border: 1px red solid; }
 
 .selector-popup {
   background-color: $bg_color_selectionpopup;
 }
+.reply-highlighted { border: 1px #FF0000 solid; }
```

Comment:
Do we need to rename all the colors? I think it's okay to use named variants?

Reply:
I don't have strong opinions but I think that was preceeding me pulling it out into a variable which we do wanna do, and I feel like it's tidier for those to use the hexes.
Idk if the linter has a reason tho